src/plugins/intel_cpu/src/nodes/subgraph.h

```diff
@@ -85,6 +85,7 @@ class Subgraph : public Node {
     std::vector<ptrdiff_t> start_offset_out = {};
 
     RepackedInputConfig repacked_constant_input_config = {};
+    std::set<size_t> external_ptrs_idces = {};
```

Comment:
Minor: maybe `external_ptrs_input_idces` + brief comment? Otherwise it's not so clear what is it 🤔

Reply:
Good idea, thanks! Done, please take a look